src/libs/SearchUtils.ts

```diff
@@ -522,6 +522,8 @@ function buildSearchQueryString(queryJSON?: SearchQueryJSON) {
  * Given object with chosen search filters builds correct query string from them
  */
 function buildQueryStringFromFilterValues(filterValues: Partial<SearchAdvancedFiltersForm>) {
+    let expenseFilter = '';
```

Comment:
```suggestion
    let typeFilter = '';
```

this is called TYPE in search.